src/nvautoinstall/MainFunction.py

```diff
@@ -112,7 +112,7 @@ def avbl(self):
         return "rpmfusion-nonfree-nvidia-driver" in output
 
     def conn(self):
-        retndata = subprocess.getstatusoutput("ping -c 3 -W 3 rpmfusion.org")[0]
+        retndata = subprocess.getstatusoutput("curl rpmfusion.org")[0]
```

Comment:
The usage with CURL returns the following output.
```
curl rpmfusion.org
```
```
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>302 Found</title>
</head><body>
<h1>Found</h1>
<p>The document has moved <a href="https://rpmfusion.org/">here</a>.</p>
<hr>
<address>Apache Server at rpmfusion.org Port 80</address>
</body></html>
```

I suggest it be changed to the following.
```suggestion
        retndata = subprocess.getstatusoutput("curl https://rpmfusion.org/")[0]
```